fe/fe-core/src/main/java/com/starrocks/server/GlobalStateMgr.java

```diff
@@ -2052,6 +2036,26 @@ private boolean canSkipBadReplayedJournal() {
             LOG.warn("failed to parse metadata_journal_skip_bad_journal_ids: {}",
                     Config.metadata_journal_skip_bad_journal_ids, e);
         }
+
+        short opCode = OperationType.OP_INVALID;
+        if (t instanceof JournalException) {
+            opCode = ((JournalException) t).getOpCode();
+        }
+        if (t instanceof JournalInconsistentException) {
+            opCode = ((JournalInconsistentException) t).getOpCode();
```

Comment:
how about `JournalInconsistentException` extends from `JournalException`?

Reply:
There are different meaning between  `JournalInconsistentException` and `JournalException`, and the processing methods are also different. Maybe we can integrate the two in the future.